### ISSUE TITLE
Malware found in the jenkins.exe

### ISSUE THREAD

**Issue by clarancedriver** [rapid7/metasploitable3#280]
## Issue Description
**Please check the [General Issues](https://github.com/rapid7/metasploitable3/wiki/General-Issues) section in the wiki before you submit the issue.**
If you didn't find your issue mentioned, please give a thorough description of the issue you're seeing.
Also, please be sure to include any troubleshooting steps that you've already attempted.

## Host System
* OS: windows
* Packer Version: 1.2.2
* Vagrant Version: 1.9.1
* VirtualBox Version: 5.1.34

## Command Output
Copy the relevant command output here.
If it's long, either post to a gist and add the link here, or isolate the error lines.
Trojan:Win32/Bitrep.B found when cloning repo.  resources\jenkins\jenkins.exe

**Comment by todb-r7** [comment]
Say @wchen-r7 , did you trojan jenkins.exe 2 years ago?

@clarancedriver , this has got to be a false positive from Windows Defender. In any event, if you're worried about it, don't run jenkins.exe.

**Comment by wchen-r7** [comment]
@clarancedriver Hi, sorry for the scare. It's a false positive. We used a 3rd party tool to create Jenskins.exe as a Windows service, and it looks like Windows Defender's cloud protection doesn't like the way it's packaged.

We actually can't remember the name of the 3rd party tool we used anymore, but after some quick reversing, we can tell it's a custom .Net application that has a Powershell "ScriptRunner" component, with the ps1 file also embedded in there. The embedded ps1 file is Base64'd and zipped. When you extract that, you will see that it's actually executing this command:

```
java -jar "C:\Program Files\jenkins\jenkins.war" --httpPort=8484
```

So this EXE is harmless.

However, I submitted this false positive to Microsoft anyway, and hope they will correct that soon:

![screen shot 2018-05-08 at 11 50 11 am](https://user-images.githubusercontent.com/1170914/39770889-feca77b8-52b5-11e8-8630-17e4ef1ecdb6.png)

We will probably rebuild this EXE from scratch so that it doesn't appear to be malicious.
